epub34/a11y/index.html

```diff
@@ -1365,6 +1365,18 @@ <h5>Evaluation date</h5>
 								href="https://www.w3.org/TR/epub/#subexpression">associated with</a> [[epub-3]] the <a
 								href="#sec-evaluator-name">evaluator's name</a>.</p>
 
+						<p>For human readability and localization purposes, the value of the property MUST be a string
+							that matches one of the following [[iso8601-1]] patterns:</p>
+
+						<dl>
+							<dt><code>YYYY-MM-DD</code></dt>
+							<dd> The year, month, and date of the evalation. This format is preferred. </dd>
```

Comment:
I think ISO 8601 refers to the DD portion as "day" or "calendar day", or even more precisely "ordinal number of a calendar day". Maybe change this to "year, month, and day" instead of "year, month, and date".